app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -3371,7 +3369,7 @@ class ChatActivity :
             CONTENT_TYPE_VOICE_MESSAGE -> message.isVoiceMessage
             CONTENT_TYPE_POLL -> message.isPoll()
             CONTENT_TYPE_LINK_PREVIEW -> message.isLinkPreview()
-            CONTENT_TYPE_SYSTEM_MESSAGE -> !TextUtils.isEmpty(message.systemMessage)
+            CONTENT_TYPE_SYSTEM_MESSAGE -> message.messageType == "system"
```

Comment:
Any idea why the previous check did not work? Because `systemMessage` should not be empty in this case?
![image](https://github.com/user-attachments/assets/8c10a537-490f-41f6-a198-b96f2ef25f0c)

Reply:
Ahh, I see, seems that the federated_user_added and federated_user_removed weren't added to the system message type enum. Should be fixed properly now.